test/functional/outputs/splunk/forward_to_splunk_test.go

```diff
@@ -44,7 +44,7 @@ var _ = Describe("Forwarding to Splunk", func() {
 		Expect(framework.Deploy()).To(BeNil())
 
 		// Wait for splunk to be ready
-		time.Sleep(45 * time.Second)
```

Comment:
Increase this to allow Splunk more time to set up. It is flaking because it takes about a minute for Splunk to be ready